content/chapters/io/lab/solution/client-server/client.c

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: BSD-3-Clause
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
+#include "utils/utils.h"
+
+#define PORT	5000
+#define BUFSIZE	1024
+
+const char *localhost = "127.0.0.1";
+
+int readInputWithPrompt(char *buf)
```

Comment:
```suggestion
int read_input_with_prompt(char *buf)
```
Same for all functions.